learn/tanssi/network-features/staking.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Staking
 description: Learn how Tanssi implements a novel Staking mechanism to provide a decentralized and trustless set of block producers to all Tanssi-powered appchains
+icon: material-pickaxe
```

Comment:
Any other options for this icon?

Reply:
i love the pickaxe, but i will look for another one haha